admin/src/pages/SettingsPage/types.ts

```diff
@@ -0,0 +1,21 @@
+import { StrapiContentTypeFullSchema } from "strapi-typed";
+import { NavigationPluginConfig } from "../../../../types";
+export type RawPayload = {
+  selectedContentTypes: string[];
+  nameFields: Record<string, string[]>;
+  audienceFieldChecked: boolean;
+  allowedLevels: number;
+  populate: Record<string, string[]>;
+  i18nEnabled: boolean;
+}
+
+// TODO: This type should be replaced with new version of content type in new strapi-typed
+export type ContentTypeToFix = StrapiContentTypeFullSchema & { available: boolean, isSingle: boolean, plugin: string }
+
+export type IPreparePayload = (payload: { form: RawPayload, pruneObsoleteI18nNavigations: boolean }) => NavigationPluginConfig;
```

Comment:
`I` in a name stands for `Interface`. I'd rather to see `export interface` :D and leave name "as is"